wwwroot/index.html

```diff
@@ -338,14 +338,21 @@ <h2 class="mb-5">Frequently Asked Questions</h2>
                             </div>
                         </div>
                     </div>
-                    <div class="col-lg-4">
+                    <div class="col-lg-5">
                         <div class="sticky-top pt-header">
-                            <div class="box" id="addtowasabi">
+                            <strong class="text-primary mb-2 d-block lh">We strongly recommend using the Ginger Wallet client for long-term reliability!</strong>
+                            <a href="#download" class="btn btn-primary mb-4">Download</a>
+                            <div class="box" id="addtobtcpay">
+                                <h4 class="mb-4">What is Ginger Coordinator address?</h4>
+                                <strong>https://api.gingerwallet.io/</strong>
+                            </div>
+
+                            <div class="box mt-4" id="addtowasabi">
                                 <h4 class="mb-4">How to add Ginger coordinator to Wasabi Wallet?</h4>
                                 <ol class="text-start">
-                                    <li>Make sure that you are using the latest Wasabi Wallet version.</li>
-                                    <li>Go to Settings -> Bitcoin tab -> Coordinator URI</li>
-                                    <li>Change https://api.wasabiwallet.io/ -> https://api.gingerwallet.io/</li>
+                                    <li>Make sure that you are using Wasabi Wallet 2.0.7.2 or earlier version.</li>
```

Comment:
Changing the coordinator in the config file, works with all the versions. So probably this line can be removed.
Chaing the coordinator in the software is only avalable in and after 2.0.8.0.

Reply:
But after 2.0.7.2 fees will be 0%. That's why we added it.

Reply:
I don't think so, they won't yet add it in 2.0.8.0 either. And I am not even sure if they even add it later.